docs/config.rst

```diff
@@ -1,11 +1,606 @@
+
+
+
+Task Configurations
+-------------------
+
+Task configurations are defined under the ``tasks`` section of your ``cumulusci.yml`` file. Each task configuration defines a task which can be run using the ``cci task run`` command, or included in a flow step. With a few simple changes to this section, you can configure build automation functionality without ever touching Python code.
+
+
+Override a Task Option
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you repeatedly  specify the same value for an option while running a task, you can configure CumulusCI to use that particular value as a default value.
+
+     Example: Let's enforce a 90% code coverage requirement for Apex code in your project. The `run_tests<TODO>`_ task, which executes all Apex Tests in a target org, can enforce code coverage at a given percentage by passing the ``--required_org_code_coverage_percent`` option.
+
+.. code-block:: yaml
+
+    run_tests:
+        options:
+            required_org_code_coverage_percent: 90
+
+When the code is placed under the ``tasks`` section of the ``cumulusci.yml`` file, CumulusCI overrides the default option with a value of ``90``. The overridden option is used whenever this automated task is called, unless it is further overridden for a particular flow step.
+
+Verify the change by looking for a default option value when running ``cci task info <name>``.
+
+.. code-block:: yaml
+
+    $ cci task info run_tests
+    run_tests
+
+    Description: Runs all apex tests
+
+    Class: cumulusci.tasks.apex.testrunner.RunApexTests
+
+    Command Syntax
+
+        $ cci task run run_tests
+
+    Options
+        .
+        .
+        .
+      -o required_org_code_coverage_percent PERCENTAGE
+        Optional
+        Require at least X percent code coverage across the org following the test run.
+        Default: 90
+
+
+Add a Custom Task
+^^^^^^^^^^^^^^^^^
+
+To define a new task for your project, simply add the name of the new task under the ``tasks`` section of your ``cumulusci.yml`` file.
+
+    Example: Let's create a custom task named ``deploy_reports`` that deploys a set of reports stored in your project's unpackaged metadata located in ``unpackaged/config/reports``.
+
+    First, look up the Python class associated with the standard task ``deploy``. From there we can see that the ``deploy`` task has a ``class_path`` value of ``cumulusci.tasks.salesforce.Deploy``.
+
+    Store the task under the ``tasks`` section of the ``cumulusci.yml`` file.
+
+.. code-block:: yaml
+
+    deploy_reports:
+        description: Deploy Reports 
+        class_path: cumulusci.tasks.salesforce.Deploy
+        group: projectName
+        options:
+            path: unpackaged/config/reports
+
+.. note:: Be sure to include the value we retrieved for ``class_path``. It's also best practice to add a common ``group`` attribute to your custom tasks in order to make it easier to see the tasks that are specific to your project when using ``cci task list``.
```

Comment:
```suggestion
.. note:: Be sure to include the value we retrieved for ``class_path``. It's also best practice to add a common ``group`` attribute to your custom tasks so that it's easier to see the tasks that are specific to your project when using ``cci task list``.
```
"in order to" entry at https://confluence.internal.salesforce.com/display/DOCTEAM/wordiness